offerImport/importHandler/importOffersToTradeshift.go

```diff
@@ -18,53 +18,53 @@ func NewImportOfferHandler(deps Deps) ImportOfferInterface {
 }
 
 func (i *ImportOfferHandler) ImportOffers(offers []offerReader.RawOffer) {
-
 	for _, item := range offers {
-		res, err := i.ImportOffer(item.Offer, item.Receiver)
-		fmt.Sprintf("This is res %v, and this is err: %v", res, err)
+		_, err := i.ImportOffer(item.Offer, item.Receiver)
+		if err != nil {
+			log.Printf("failed to import offer \"%v\". Reason:  %v", item.Offer, err)
+		}
 	}
 }
 
 func (i *ImportOfferHandler) ImportOffer(offerName string, buyerID string) (Status, error) {
-
-	err := i.findBuyer(offerName, buyerID)
+	err := i.findBuyer(buyerID)
 	if err != nil {
 		return BuyerNotFound, err
 	}
 
-	isFound, err := i.findOffer(offerName, buyerID)
+	isFound, err := i.findOfferByName(offerName, buyerID)
 	if err != nil {
 		return Failed, err
 	}
 	if isFound {
-		//todo update offer (mvp 2)
+		log.Print("offer \"%v\" was found", offerName)
 		return OfferFound, nil
 	}
 
 	code, err := i.createOffer(offerName, buyerID)
 	if err != nil {
-		return Failed, fmt.Errorf("failed to create offer: %v", err)
+		return Failed, fmt.Errorf("offer was not created: %v", err)
 	}
 
-	log.Println("new offer created: %v", code)
+	log.Printf("new offer was created: \"%v\"", code)
 	return OfferCreated, nil
 }
 
-func (i *ImportOfferHandler) findBuyer(offerName string, buyerID string) error {
+func (i *ImportOfferHandler) findBuyer(buyerID string) error {
```

Comment:
This call looks weird: method findBuyer doesn't find anything. I suggest to rename method to something more meaningfull. Also it can be good idea to handle servers errors (5xx) and 404 (not found) in a different way.

Reply:
renamed method+changed logic + added error-message bodyes to RST response in case of not 200 response as fast solution